codebuild_specs/aggregate_e2e_reports.yml

```diff
@@ -4,7 +4,12 @@ env:
 phases:
   build:
     commands:
-      - source ./shared-scripts.sh && _waitForJobs $CODEBUILD_SOURCE_VERSION $WAIT_FOR_IDS
+      - cd ./scripts
+      - yarn install
+      - npm install -g ts-node
+      - npm install aws-sdk
```

Comment:
I think yarn install will cover these dependencies and then you can invoke ts-node on the line below with `yarn ts-node`

Reply:
I found these two lines to be necessary. Without them, it failed.

Reply:
What were the errors? I'm not sure if npm will respect the locked dependencies in `yarn.lock` during install, but if it doesn't then this can break if either dependency has a bad minor or patch release.

Reply:
The error was that it could not resolve ts-node and/or aws-sdk. Doing an npm install for the packages alleviated this problem.

Reply:
@lazpavel this might be worth revisiting during the yarn2 migration, but I think it's fine the way it is now.